### ISSUE TITLE
Research animation library

### ISSUE THREAD

**Issue by AtaReversei** [basliq/luna-web#17]
We need an animation library in our project. We want it to be free, performant, compatible with TS and easy to use without a verbose syntax

**Comment by AtaReversei** [comment]
Out of 3 or 4 different `React` or `JS` libraries I found `Framer Motion` better than others. It has a ton of documentation, tutorials, videos and examples. I just didn't find any proof for `Typescript` support but I guess it will be fine.
The other library I think we will need other than `Framer Motion` is `Confetti Cannon` ([link](https://github.com/herrethan/react-confetti-explosion)). It is not easy to create a confetti explosion with proper physics.
So to sum up:

- Framer Motion
- React Confetti Explosion
